### PR TITLE
Add README.md and GETTING_STARTED.md.

### PR DESCRIPTION
**Pull request by qingqing01**
For convenience of review, see  https://github.com/qingqing01/models/tree/docs/PaddleCV/object_detection

### CODE REVIEW

File: PaddleCV/object_detection/README.md [hunk 1]
```diff
@@ -1,4 +1,130 @@
-PaddlePaddle Object Detection
-===
+# PaddlePaddle Object Detection
 
-The document will be coming soon.
+This object detection framework is based on PaddlePaddle. We want to provide classical and state of the art detection algorithms in generic object detection and specific target detection for uers. And we aimed to make this framework easy to extend, train and depoly uers' task. We aimed to make it easy to use in research and products.
```
Comment (jerrywgz) [inline PaddleCV/object_detection/README.md]: uers->users

File: PaddleCV/object_detection/README.md [hunk 2]
```diff
@@ -1,4 +1,130 @@
...
+
+## Introduction
+
+Major features:
+
+- Easy to Depoly:
+  All the operations related with inference are implemented by C++ and CUDA, it makes the detection model easy to depoly in products on server without Python based on the high efficient inference engine of PaddlePaddle.
+  We release detection models based on ResNet-vd backbone, for example, the accuracy of Faster RCNN model with FPN based on ResNet50 VD is close to model based on ResNet 101. But the former is more smaller and faster.
```
Comment (jerrywgz) [inline PaddleCV/object_detection/README.md]: more smaller->smaller

File: PaddleCV/object_detection/README.md [hunk 3]
```diff
@@ -1,4 +1,130 @@
...
+
+- Easy to Customize:
+   All components are modular encapsulated, including the data transforms. It's easy to plug in and pull out any module. For example, uers can switch backone easily or add data mixed-up data transform for models.
```
Comment (jerrywgz) [inline PaddleCV/object_detection/README.md]: backone->backbone

Comment (jerrywgz) [reply PaddleCV/object_detection/README.md]: data mixed-up data transform ->mixup data augmentation

File: PaddleCV/object_detection/docs/GETTING_STARTED.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+#### Multi-Device Training
+
+```bash
+export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 # set devices
+# export CPU_NUM=8 # for CPU training
+python tools/train.py -c =configs/faster_rcnn_r50_1x.yml
+```
+
+- Default dataset directory is `dataset/coco`, users also can specify it in the configure file.
+- Pretrained model will be downloaded automatically.
```
Comment (jerrywgz) [inline PaddleCV/object_detection/docs/GETTING_STARTED.md]: 是否需要告诉用户自动下载的目录

File: PaddleCV/object_detection/docs/GETTING_STARTED.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+- Infer several images:
+
+```bash
+export CUDA_VISIBLE_DEVICES=0
+python tools/infer.py -c configs/faster_rcnn_r50_1x.yml --infer_dir=demo
+```
+
+The predicted and visualized images are saved in `output` by default, uers also can change the saved directory by specifying `--savefile=`.  For more help please run `python tools/infer.py --helo`.
```
Comment (jerrywgz) [inline PaddleCV/object_detection/docs/GETTING_STARTED.md]: helo->help

File: PaddleCV/object_detection/README.md [hunk 6]
```diff
@@ -1,4 +1,130 @@
...
+| Cascade R-CNN      | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| RetinaNet          | ✓      | ✓      | ✓        |  ✓       | ✗        | ✗      |
+| Yolov3             | ✓      | ✗      | ✗         |  ✗       | ✓        | ✓     |
+| SSD                | ✗      | ✗      | ✗         |  ✗       | ✓        | ✗      |
+
+The extensive capabilities:
+
+- [x] **Synchronized batch norm**:  used in Yolo v3.
+- [x] **Group Norm**: supported this operation, the related model will be added later.
+- [x] **Dodulated deformable convolution**: suppored this operation, the related model will be added later.
```
Comment (willthefrog) [inline PaddleCV/object_detection/README.md]: `Modulated`?

File: PaddleCV/object_detection/README.md [hunk 7]
```diff
@@ -1,4 +1,130 @@
...
+
+
+## Deploy
+
+The example how to use PaddlePaddle to deploy detection model will be added later.
+
+
+## Framework
+
+The code tree is as follows.
```
Comment (willthefrog) [inline PaddleCV/object_detection/README.md]: IMHO, this is should not be included for 2 reasons
- folder names are quite self explanatory 
- the directory structure is still subject to future change

File: PaddleCV/object_detection/docs/GETTING_STARTED.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+- Default dataset directory is `dataset/coco`, users also can specify it in the configure file.
+- Pretrained model will be downloaded automatically.
+- Model will be saved at  `output/faster_rcnn_r50_1x` by default, users also can specify it in the configure file.
+- All hyper parameters can refer input config.
+- Change config file for other models.
+- For more help, please run `python tools/train.py --help`.
+
+
+For `SSD` on Pascal-VOC dataset,  set `--eval=True` to do evaluation during training.
+For other models based on COCO dataset, the evaluating during training is not fully verified, better to do evaluation after traning.
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/GETTING_STARTED.md]: `traning` -> `training`

File: PaddleCV/object_detection/docs/GETTING_STARTED.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+
+### Distributed Training
```
Comment (willthefrog) [inline PaddleCV/object_detection/docs/GETTING_STARTED.md]: leave this section out and add back when distributed training is in place?

**Comment by willthefrog** [comment]
Oh, boy, this one needs some serious revision😂.  Doing this over the review process would be too tedious with all the wording/phrasing issues, I suggest just merge this for now, and please remind me to proofread it again and submit a separate PR.

File: PaddleCV/object_detection/README.md (hunk 2 above)
Comment (qingqing01) [reply PaddleCV/object_detection/README.md]: Done

File: PaddleCV/object_detection/README.md (hunk 3 above)
Comment (qingqing01) [reply PaddleCV/object_detection/README.md]: Done

File: PaddleCV/object_detection/README.md (hunk 7 above)
Comment (qingqing01) [reply PaddleCV/object_detection/README.md]: Removed

File: PaddleCV/object_detection/docs/GETTING_STARTED.md (hunk 4 above)
Comment (qingqing01) [reply PaddleCV/object_detection/docs/GETTING_STARTED.md]: Done

File: PaddleCV/object_detection/docs/GETTING_STARTED.md (hunk 9 above)
Comment (qingqing01) [reply PaddleCV/object_detection/docs/GETTING_STARTED.md]: Done

File: PaddleCV/object_detection/docs/GETTING_STARTED.md (hunk 8 above)
Comment (qingqing01) [reply PaddleCV/object_detection/docs/GETTING_STARTED.md]: Done

File: PaddleCV/object_detection/docs/GETTING_STARTED.md (hunk 5 above)
Comment (qingqing01) [reply PaddleCV/object_detection/docs/GETTING_STARTED.md]: Done

File: PaddleCV/object_detection/README.md (hunk 6 above)
Comment (qingqing01) [reply PaddleCV/object_detection/README.md]: Done

**Review by willthefrog — APPROVED** [review]
